src/main/groovy/com/deploygate/gradle/plugins/tasks/LoginTask.java

```diff
@@ -0,0 +1,230 @@
+package com.deploygate.gradle.plugins.tasks;
```

Comment:
Translated into Java from Groovy because the most of code are changed by refactoring.